docs/bias_correction.rst

```diff
@@ -1,22 +1,84 @@
-Bias Correction **(IN PROGRESS)**
-=================================
+Bias Correction
+===============
 
 This page documents the implementation of
 **bmorph** bias correction for streamflow
 data in a watershed. An example workflow notebook
-can be found in *example_notebooks*.
+can be found in ``bmorph_tutorial``.
+
+bmorph Overview
+---------------
+
+                       +-------------------------+-----------------+
+                       | Spatially Consistent BC | Independent BC  |
+                       |         (SCBC)          |      (IBC)      |
++----------------------+-------------------------+-----------------+
+| Unconditional BC (U) |         SCBC_U          |      IBC_U      |
++----------------------+-------------------------+-----------------+
+|  Conditional BC (C)  |         SCBC_C          |      IBC_C      |
++----------------------+-------------------------+-----------------+
 
-bmorph Usage
--------------------
 Independent Bias Correction: Univariate (IBC_U)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Univariated Independent Bias Correction (IBC_U) is considered the traditional bias correction method implemented here as described in `EDCDFm`_. This method can only be performed at sites with refernce data, which is useful when gauge sites can measure flows but does not gaurentee spatially consitent corrections amongst a series of gauge sites. None the less, it performs best out of all four methods at the gauge sites where it can correct, but may introduce spatial incongruties across sites.
+
+`Callable workflow`_ : ``apply_annual_bmorph``, ``apply_interval_bmorph``
```

Comment:
The phrase `Callable workflow` sounds a bit obscure. Perhaps change this to `Workflow functions` or something similar?